bin/epaa.py

```diff
@@ -875,6 +917,35 @@ def generate_wt_seqs(peptides):
     return wt_dict
 
 
+def create_peptide_variant_dictionary(peptides):
```

Comment:
As already discussed, it might be easier to do some minor changes in the Peptide class of epytope in order to get the information of the two following functions. Maybe we can add a #TODO here